### PR TITLE
Added liveupdate.get_stored_resources()

### PR DESCRIPTION
**Pull request by JCash**

### CODE REVIEW

File: engine/gamesys/src/gamesys/scripts/script_collectionproxy.cpp [hunk 1]
```diff
@@ -180,7 +186,8 @@ namespace dmGameSystem
 
     static const luaL_reg Module_methods[] =
     {
-        {"missing_resources", CollectionProxy_MissingResources},
+        {"missing_resources", Deprecated_CollectionProxy_GetMissingResources},
+        {"get_missing_resources", CollectionProxy_GetMissingResources},
```
Comment (JCash) [inline engine/gamesys/src/gamesys/scripts/script_collectionproxy.cpp:190]: It felt wrong to add more "get_" functions, and at the same time leave this behind.

File: engine/gamesys/src/gamesys/scripts/script_resource_liveupdate.cpp [hunk 2]
```diff
@@ -274,6 +274,57 @@ namespace dmLiveUpdate
         return 0;
     }
 
+    /*# get a list of stored resources in the dynamic archive
+     *
+     * Get a list of stored resources in the dynamic archive. These resource have previously been
+     * stored using liveupdate.store_resource()
+     *
+     * @name resource.get_stored_resourced
```
Comment (britzl) [inline engine/gamesys/src/gamesys/scripts/script_resource_liveupdate.cpp:282]: get_stored_resourced -> get_stored_resources

**Review by britzl — DISMISSED** [review]
Looks good. Found one typo in API docs.

**Review by britzl — APPROVED** [review]

**Comment by JCash** [comment]
Closing for now.

**Pull request closed without merging**